l10n_jp_address_layout/views/assets.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="0">
+        <template id="assets_common" inherit_id="web.assets_backend">
```

Comment:
It'd be more appropriate to name template id `assets_backend` instead of `assets_common`.

Reply:
Sorry, It's my typo!
I fixed the template id.

And I removed the editorconfig file and added my self to the list of contribution.
I'll try to sign OCA CLA.